package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nested-property",
   "description": "Read, write or test a data structure's nested property via a string like 'my.nested.property'. It works through arrays and objects.'",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
let's actually bump to 1.0.0, I would think that the change is a breaking change since it changes the API, maybe some libs are relying on the fact that it creates an object instead of an array (can't remember how I used it :))

Reply:
It's true! I had doesn't think in this!